src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config.jelly

```diff
@@ -47,6 +47,10 @@
         <f:textbox default="10"/>
     </f:entry>
 
+    <f:entry title="${%Max number of current connections}" field="maxRequestsPerHostStr">
```

Comment:
It would be nice to have a more detailed description of what this does on the configuration page. Or maybe changing the title to `Max current connections to Kubernetes API` would be sufficient.